apps/io_tester/io_tester.cc

```diff
@@ -530,7 +530,7 @@ class io_class_data : public class_data {
                             return max_concurrent_for_each(pos.begin(), pos.end(), 64, [this, bufsize] (auto pos) mutable {
                                 auto bufptr = allocate_aligned_buffer<char>(bufsize, 4096);
                                 auto buf = bufptr.get();
-                                std::uniform_int_distribution<char> fill('@', '~');
+                                std::uniform_int_distribution<signed char> fill('@', '~');
```

Comment:
```suggestion
                                std::uniform_int_distribution<int> fill('@', '~');
```
as `memset()` takes `int` as its second parameter.